### ISSUE TITLE
[Bug]: no_proxy error behind corporate firewall

### ISSUE THREAD

**Issue by sarathsurpur** [BerriAI/litellm#3556]
### What happened?

A bug happened!

As discussed in https://github.com/BerriAI/litellm/issues/2127 I am encountering issue when accessing routing object in litellm, 
I tried updating no_proxy env variable with recommended changes
**error: ValueError: Proxy keys should use proper URL forms rather than plain scheme strings. Instead of "x.x.x.x", use "x.x.x.x://"**

code 👍 
```
from litellm import Router
router = Router(model_list=model_list, set_verbose=False)```




### Relevant log output

_No response_

### Twitter / LinkedIn details

_No response_

**Comment by ghost** [comment]
Hey @sarathsurpur have a PR out for this, hoping to have this fixed soon. 

Can you check if my test covers your scenario - https://github.com/BerriAI/litellm/pull/4434/files#diff-e4391c9023696dfb1c5cc812d142ae613babbfacd4b9c6085b043d467ab3a90b

**Comment by sarathsurpur** [comment]
@krrishdholakia  is it already updated in the library to handle this, jus checking with you?

**Comment by ghost** [comment]
hey @sarathsurpur not live yet - hoping to have it live by midweek (wednesday night) 

will be working more on it tomorrow

**Comment by ghost** [comment]
Hey @sarathsurpur put out a dev release with fix. Can you confirm it works for you - https://github.com/BerriAI/litellm/releases/tag/v1.41.3.dev3

**Comment by sarathsurpur** [comment]
looks like still the issue is occurring, installed version 1.41.3

ValueError: Proxy keys should use proper URL forms rather than plain scheme strings. Instead of "xx.x.x.x" use "xx.x.x.x://"

**Comment by ghost** [comment]
hey @sarathsurpur the fix is on `v1.41.3.dev3` 

can you confirm that's the one you installed
